CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
-## [Unreleased]
```

Comment:
It's typically a good idea to keep this line in, but commented out, to signify to the next developer they should put their changes under that section.

Reply:
Good idea, thanks 👍